bin/build-db

```diff
@@ -11,8 +11,16 @@ console.log(`Building database for the ${process.env.NODE_ENV} env…`);
 
 let foodProcesses, textileProcesses;
 if (process.env.NODE_ENV === "test") {
-  foodProcesses = "public/data/food/processes_impacts.json";
-  textileProcesses = "public/data/textile/processes_impacts.json";
+  foodProcesses = process.env.FOOD_PROCESSES_IMPACTS_PATH;
+  textileProcesses = process.env.TEXTILE_PROCESSES_IMPACTS_PATH;
+
+  if (!foodProcesses || !textileProcesses) {
+    console.error(
+      "\n🚨 ERROR: For the tests to work properly, you need to specify FOOD_PROCESSES_IMPACTS_PATH and TEXTILE_PROCESSES_IMPACTS_PATH env variables. They need to point to the detailed versions of the processes files. Please, edit your .env file accordingly.",
+    );
```

Comment:
We might benefit from template strings to format this long line over multiple ones for legibility here